git-coco

```diff
@@ -184,6 +209,9 @@ case "$command" in
     pr)
         pr "$@" || exit 1
         ;;
+    tag)
+        tag "$@" || exit 1
+        ;;
     *)
         fail "Usage: git coco ls|create|clone|rm|init|pr"
```

Comment:
The usage information should also be updated

Reply:
fixed!